plugins/PRAplugin/src/MCSSolver.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on June 24, 2020
+
+@author: mandd
+"""
+
+from __future__ import division, print_function , unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default', DeprecationWarning)
+
+#External Modules---------------------------------------------------------------
+import numpy as np
+import itertools
+import math
+#External Modules End-----------------------------------------------------------
+
+#Internal Modules---------------------------------------------------------------
+from PluginsBaseClasses.ExternalModelPluginBase import ExternalModelPluginBase
+from PostProcessors.MCSimporter import MCSreader
+#Internal Modules End-----------------------------------------------------------
+
+class MCSSolver(ExternalModelPluginBase):
+  """
+    This class is designed to create a MCS solver model
+  """
+  def __init__(self):
+    """
+      Constructor
+      @ In, None
+      @ Out, None
+    """
+    ExternalModelPluginBase.__init__(self)
+
+  def _readMoreXML(self, container, xmlNode):
+    """
+      Method to read the portion of the XML that belongs to the MCS solver model
+      @ In, container, object, self-like object where all the variables can be stored
+      @ In, xmlNode, xml.etree.ElementTree.Element, XML node that needs to be read
+      @ Out, None
+    """
+    container.filename = None
+    container.mapping    = {}
+    container.InvMapping = {}
+
+    for child in xmlNode:
+      if child.tag == 'topEventID':
+        container.topEventID = child.text.strip()
+      elif child.tag == 'solverOrder':
+        self.solverOrder = int(child.text.strip())
```

Comment:
you may have another error if the user provided is not integer or float value.

Reply:
added try except